apps/test/unit/lib/levelbuilder/lesson-editor/ActivityCardTest.jsx

```diff
@@ -36,8 +36,8 @@ describe('ActivityCard', () => {
 
   it('renders default props', () => {
     const wrapper = shallow(<ActivityCard {...defaultProps} />);
-    expect(wrapper.contains('Activity:'));
-    expect(wrapper.contains('Time (mins):'));
```

Comment:
without `.to.be.true`, these calls weren't doing anything, which I noticed because "Time (mins)" is not exactly how it appears in the UI. I'll fix a few more in the next PR: https://github.com/code-dot-org/code-dot-org/pull/36915

Reply:
Oh wow well thats good to know.